README.md

```diff
@@ -0,0 +1,64 @@
+# Diplomat
+
+Diplomat is a conversational agent framework for goal-oriented group discussion. Recent work in human-computer interaction has explored the use of conversational agents as facilitators for group goal-oriented discussions. Inspired by this work and by the apparent lack of tooling available to support it, we created Diplomat, a Python-based framework for building conversational agent facilitators. Diplomat is designed to support simple specification of agent functionality as well as customizable integration with online chat services. We document a preliminary user study we conducted to help inform the design of Diplomat.
+
+## How to run Diplomat? 
+Diplomat has currently been configured to run seamlessly with Slack. 
```

Comment:
I would say that Diplomat provides a default configuration with Slack. There are still some steps a user has to take to get it working, so it's a little strong to say it's "seamless".